test/journeys/specs/recents/dataApi/space-type-filter.js

```diff
@@ -0,0 +1,186 @@
+import {assert} from 'chai';
```

Comment:
Aren't these tests in the space-list-filter file now?

Reply:
forgot to remove this file...